servers/Fabric.Mcp.Server/src/Fabric.Mcp.Server.csproj

```diff
@@ -5,17 +5,37 @@
     <AssemblyTitle>Fabric MCP Server</AssemblyTitle>
     <Description>Microsoft Fabric MCP Server - Model Context Protocol implementation for Fabric</Description>
     <ReadmeUrl>https://github.com/Microsoft/mcp/blob/main/servers/Fabric.Mcp.Server#readme</ReadmeUrl>
+    <ReadmePath>$(RepoRoot)/servers/Fabric.Mcp.Server/README.md</ReadmePath>
+
+    <!--
+    Defaults to the microsoft logo if not specified.
+    <PackageIcon>$(RepoRoot)/servers/Fabric.Mcp.Server/fabric_icon.png</PackageIcon>
+    -->
+
+    <!-- npm settings -->
     <NpmPackageName>@microsoft/fabric-mcp</NpmPackageName>
     <NpmPackageKeywords>fabric,mcp,model-context-protocol</NpmPackageKeywords>
-    <PackageId>Microsoft.Fabric.Mcp</PackageId>
-    <ToolCommandName>$(CliName)</ToolCommandName>
-    <AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
+
+    <!-- docker settings -->
+    <DockerImageName></DockerImageName>
```

Comment:
Intentionally empty?

Reply:
It is.  We need to sync with the fabric team on their image name.  Their releases will fail until we get that.